MAUI/DataForm/Validation.md

```diff
@@ -15,6 +15,177 @@ The data form validates the data and user input to update the correct value in t
 
 The supported built in validations are as follows:
 
+#### IDataErrorInfo
+You can validate the data by implementing the `IDataErrorInfo` interface in the data object class.
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : IDataErrorInfo, INotifyPropertyChanged
```

Comment:
check INotifyPropertyChanged is required, if not remove related proeprties

Reply:
INotifyPropertyChanged removed and updated in property also